packages/hydrogen-edge-adapter/README.md

```diff
@@ -0,0 +1,60 @@
+# Hydrogen Remix Adapter for Netlify
+
+The Hydrogen Remix Adapter for Netlify allows you to deploy your [Shopify Hydrogen](https://hydrogen.shopify.dev/) (via
```

Comment:
@codebyuma, this looks good to me, but do you have any suggestions for improving the README?

Reply:
@nickytonline I agree, this content looks good to me too. But, I think there's more content on the actual docs page that we'll need to update (along with adding this content) and that's where I could use some guidance on what needs to change and how: https://docs.netlify.com/integrations/frameworks/hydrogen/